package.json

```diff
@@ -81,9 +82,9 @@
     "templates:dev": "gulp templates",
     "templates:build": "gulp templates",
     "templates:watch": "onchange app/\\*\\*/\\*.html -- npm run templates:dev",
-    "config:dev": "gulp config:dev",
-    "config:staging": "gulp config:staging",
-    "config:build": "gulp config:build",
-    "config:watch": "onchange config/config.json -- npm-run-all -s config:dev js:dev"
+    "config:dev": "ln-patterns && gulp config:dev",
+    "config:staging": "ln-patterns && gulp config:staging",
+    "config:build": "ln-patterns && gulp config:build",
```

Comment:
Do we need this in the prod build?
Also should this go in the config scripts really?